src/scripts.js

```diff
@@ -7,184 +7,225 @@ import {travelersData, destinationsData, tripsData, addNewTrip, fetchData, trips
+};
 
 function buildFormHelper() {
-  preventPickingPastDates()
-  buildDropDown()
-  displayDestinationOptions(allDestinations.destinationsRepo)
-}
+  preventPickingPastDates();
+  buildDropDown();
+  displayDestinationOptions(allDestinations.destinationsRepo);
+};
 
 //DISPLAY ALL TRIPS
 
 function displayAllTrips(tripsPerTraveler) {
-  console.log('tripsPerTaveler,', tripsPerTraveler)
   let tripsData = ''
   tripsPerTraveler.forEach((trip) => {
-    const tripDestination = allDestinations.findDestination(trip)
-    const tripTotalCost = allTrips.totalCostPerTrip(trip, tripDestination)
-    const endDate = allTrips.getEndDate(trip)
-    const tripTimeEra = allTrips.checkTripTimeEra(trip)
+    const tripDestination = allDestinations.findDestination(trip);
+    const tripTotalCost = allTrips.totalCostPerTrip(trip, tripDestination);
+    const endDate = allTrips.getEndDate(trip);
+    const tripTimeEra = allTrips.checkTripTimeEra(trip);
     
     tripsData += 
     `<section class="card">
-    <div class="photo">
-    <div class="status-label ${tripTimeEra}">
-    <p>${tripTimeEra}</p>
-    </div>
-    <img class="trip-img" src=${tripDestination.image} alt=${tripDestination.alt} width="300" height="205">
-    </div>
-    <div class="trip-info">
-    <p class="destination">${tripDestination.destination}</p>
-    <p>${trip.date} - ${endDate}</p>
-    <p>Estimated Cost: $${tripTotalCost}</p>
-    <p>Guests: ${trip.travelers}</p>
-    </div>
+      <div class="photo">
+        <div class="status-label ${tripTimeEra}">
+        <p>${tripTimeEra}</p>
+        </div>
+        <img class="trip-img" src=${tripDestination.image} alt=${tripDestination.alt} width="300" height="205">
+      </div>
+      <div class="trip-info">
+        <p class="destination">${tripDestination.destination}</p>
+        <p>${trip.date} - ${endDate}</p>
+        <p>Estimated Cost: $${tripTotalCost}</p>
+        <p>Guests: ${trip.travelers}</p>
+      </div>
     </section>`
-  })
+  });
   
-  tripsDisplay.innerHTML = tripsData
-}
+  tripsDisplay.innerHTML = tripsData;
+};
 
 function displayTotalCostThisYear(totalCost) {
-  costThisYear.innerText = `$${totalCost}`
-}
+  costThisYear.innerText = `$${totalCost}`;
+};
 
 //FORM 
 
 function displayDestinationOptions(destinations) {
-  let options = ''
+  let options = '';
   destinations.forEach(destination => {
     options += 
     `<option value="${destination.id}">${destination.destination}</option>`
-  })
-  destinationsPicker.innerHTML += options
-}
+  });
+  destinationsPicker.innerHTML += options;
+};
 
 function preventPickingPastDates() {
-  today = dayjs(Date()).format('YYYY-MM-DD')
-  datePicker.min = dayjs(today).add(1, 'day').format('YYYY-MM-DD')
-}
+  today = dayjs(Date()).format('YYYY-MM-DD');
+  datePicker.min = dayjs(today).add(1, 'day').format('YYYY-MM-DD');
+};
 
 function buildDropDown() {
-  let numbers = ''
+  let numbers = '';
   for (let i = 1; i < 51; i++) {
```

Comment:
I really like the dropdowns on your site, again great example of making the app as easy to use as possible for the user